src/utils.py

```diff
@@ -69,3 +79,39 @@ def print_os_environment():
     """
     for key, val in os.environ.items():
         print(f"{key}: {val}")
+
+
+def semver_labels_exist_in_pull_labels(pull):
```

Comment:
get here labels too, let's reduce the calls

Reply:
Will be addressed when initial_assignments conolidation is addressed separately.